media/css/mozorg/contact-spaces.scss

```diff
@@ -53,33 +51,14 @@ $image-path: '/media/protocol/img';
 }
 
 /* -------------------------------------------------------------------------- */
-// Large map
-
-.map {
-    background: $color-marketing-gray-20;
-    display: none;
-    position: relative;
-
-    img {
-        display: none;
-    }
-
-    small {
-        @include text-body-xs;
-        @include bidi(((right, 0, left, auto),));
-        background: rgba(255, 255, 255, 0.7);
-        bottom: 0;
-        padding: 0 $spacing-xs;
-        position: absolute;
-    }
-
-    @media #{$mq-md} {
-        display: block;
-
-        img {
-            display: block;
-        }
-    }
+// Hero image
+
+.hero-image {
+    background-image: url('/media/img/contact/osm/spaces/toronto-office.jpeg');
+    background-size: cover;
+    height: 450px;
```

Comment:
**issue:** the fixed height image here is causing some resize issues, and on mobile it's left with some empty space below the image creating a gap in the content.

Rather than using a background image in CSS here, might it be better to use a responsive image which is a bit more performant, and we could overlay the navigation tabs like in the same way the current page works?